autoload/SpaceVim/layers/tools.vim

```diff
@@ -59,6 +69,8 @@ function! SpaceVim#layers#tools#config() abort
   augroup rainbow_lisp
     autocmd!
     autocmd FileType lisp,clojure,scheme,java RainbowParentheses
+    autocmd FileType vimcalc setlocal nonu nornu nofoldenable | imap <buffer> <c-d> :q<cr>
+          \ | nmap <buffer> q :bdelete<cr>
```

Comment:
here you need to use `nnoremap` and `inoremap`, if someone map any char in `:bdelete<cr>`, this mapping will not work.